LinuxUtils.cpp

```diff
@@ -279,54 +216,16 @@ namespace Utils
         	return output;
 	};
 
-	bool TryCreateFolder(const std::string& path)
+    bool TryCreateFolder(const std::string& path)
 	{
-		using namespace std;
-		const char *input_begin = path.c_str();
-        	const char *input_end = input_begin+path.size();
-        	char *buffer = new char[path.size()+1];
-        	char *pos = buffer;
-        	while(input_begin !=input_end)
-		{
-                	char *next_pos = CopyFolderPathElement(input_begin, input_end, pos);
-                	if(next_pos != pos){
-                        	*next_pos = '\0';
-                        	if(!TryCreateFolderNonRecursive(buffer)){
-                                	LOG(LogLevel::Error) << "unable to create folder for path " << buffer;
-                                	delete[] buffer;
-                                	return false;
-                        	}
-                	}
-                	*next_pos = '/';
-                	++next_pos;
-                	pos = next_pos;
-        	}
-        	delete[] buffer;
-        	return true;
+		if (fs::exists(fs::u8path(path))) return true;
+		fs::create_directory(fs::u8path(path));
```

Comment:
This should probably use create_directories() so that the complete new set of directories can be done at once, rather than having to remember to create each subdirectory only after the higher-level ones.